src/openrct2/peep/Guest.cpp

```diff
@@ -6390,7 +6390,7 @@ bool loc_690FD0(Peep* peep, uint8_t* rideToView, uint8_t* rideSeatToView, TileEl
         *rideSeatToView = 1;
         if (ride->status != RIDE_STATUS_OPEN)
         {
-            if (tileElement->clearance_height > peep->next_z + 8)
+            if (tileElement->GetClearanceZ() > peep->NextLoc.z + 8 * COORDS_Z_STEP)
```

Comment:
```suggestion
            if (tileElement->GetClearanceZ() > peep->NextLoc.z + (8 * COORDS_Z_STEP))
```

Might be a bit easier to read like this. (Optional)